Testing/ContinuousIntegration/AzurePipelinesWindowsPython.yml

```diff
@@ -57,6 +57,7 @@ jobs:
           ITK_WRAP_PYTHON:BOOL=ON
           ITK_BUILD_DEFAULT_MODULES:BOOL=OFF
           ITKGroup_Core:BOOL=ON
+          ITKGroup_Filtering:BOOL=ON
```

Comment:
You don't have to enable the whole group. You can do `Module_ITKFFT` only, if that is the point.

Reply:
Fixed!